js/script.js

```diff
@@ -0,0 +1,111 @@
+var Click = (function() {
+  this.numClicks = 0;
+  this.arr = [];
+});
+
+var clickTd = new Click();
+
+//Start With click
+$('body').on('click', 'td', (function(e) {
+  var _this = this;
+  var tdClass = $(_this).children().attr('class');
+  var child = $(_this).children();
+  $(child).show();
+  setTimeout(function() {
+    clickTd.incrementClicks();
+    clickTd.addToArr(tdClass);
+  }, 500);
+}));
+
+Click.prototype.incrementClicks = function() {
+  this.numClicks++;
+  $('.clickCount').empty();
+  $('.clickCount').append("Clicks: " + this.numClicks);
+};
+
+Click.prototype.addToArr = function(tdClass) {
+  this.arr.push(tdClass);
+  if ((this.numClicks % 2) === 0) {
+    clickTd.checkGame();
+  }
+};
+
+Click.prototype.checkGame = function() {
+  if (this.arr[this.arr.length - 1] === this.arr[this.arr.length - 2]) {
+    clickTd.removePiece();
+  } else {
+    clickTd.resetPiece();
+  }
+};
+Click.prototype.resetPiece = function() {
+  $('.status').empty();
+  // $('.status').append('<p>Sorry Wrong</p>');
+  $('.' + this.arr[this.arr.length - 1]).hide();
+  $('.' + this.arr[this.arr.length - 2]).hide();
```

Comment:
What do those values you're concatenating to the period correspond to? I would use variables (or methods) to make it more obvious.